wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/button/CommandGenericHID.java

```diff
@@ -260,6 +261,74 @@ public Trigger axisGreaterThan(int axis, double threshold, EventLoop loop) {
         Pair.of(axis, threshold), k -> new Trigger(loop, () -> getRawAxis(axis) > threshold));
   }
 
+  /**
+   * Constructs a Trigger instance that is true when the axis value is active (non-zero) given a
+   * {@code deadband}, attached to the given loop.
+   *
+   * @param axis The axis to read, starting at 0
+   * @param deadband The value used to deadband the axis value. The trigger returns true if the
+   *     deadbanded value is non-zero.
+   * @param loop the event loop instance to attach the trigger to.
+   * @return a Trigger instance that is true when the deadbanded axis value is active (non-zero).
+   */
+  public Trigger axisActive(int axis, double deadband, EventLoop loop) {
+    var cache = m_axisGreaterThanCache.computeIfAbsent(loop, k -> new HashMap<>());
+    return cache.computeIfAbsent(
+        Pair.of(axis, deadband),
+        k -> new Trigger(loop, () -> MathUtil.applyDeadband(getRawAxis(axis), deadband) != 0.0));
```

Comment:
Just a suggestion:
```suggestion
        k -> new Trigger(loop, () -> Math.abs(getRawAxis(axis)) > deadband));
```
Personally, I find this at least as readable, and it avoids having to handle additional complexity in the cases where the axis *is* outside of the deadband.